rdflib/extras/infixowl.py

```diff
@@ -841,7 +894,7 @@ def deepClearIfBNode(_class):  # noqa: N802
         )
 
 
-class MalformedClass(Exception):
+class MalformedClassError(Exception):
```

Comment:
This should be inhering from ValueError
```suggestion
class MalformedClassError(ValueError):
```

Reply:
Also this makes it a breaking change.

Reply:
The change was prompted by flake8 recommendation, maybe it's not worth making the change if it's a breaking one. Might as well revert it and re-do later for 0.7, what d'you reckon?

Reply:
I will add the old class back in the type hierarchy and mark it as deprecated.

Reply:
Done now.